src/core/chain.ts

```diff
@@ -294,7 +294,7 @@
             chainsByLayer[layer].sort((a: IChain, b: IChain) => { return b.pathLength - a.pathLength });
 
             if (opts.contain) {
-                var containedChains = getContainment(chainsByLayer[layer], opts.contain);
+                var containedChains = getContainment(chainsByLayer[layer], {alernateWindings:true});
```

Comment:
The mixed type `boolean | IContainChainsOptions` was for backwards compat. But I see that `getContainment` wants only the object. Maybe we add a local var to make sure that we cast only the object as value? And I suppose the default would be `false`.

Nice catch BTW